install/bootstrap_livecd_usb.md

```diff
@@ -569,7 +563,7 @@ Now that the configuration is generated, the LiveCD must be populated with the g
 1. Unmount the data partition:
```

Comment:
I don't think it would have.   They are and were set in the typescript session.   As soon as we type "exit" they go away.   I think it was the same issue despite all of the other changes.

Reply:
PITDATA would have been defined again (from /etc/environment) once the PIT was rebooted.  But these two commands (the cp and the umount) would not have it set since they are before the boot.